src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -27,17 +27,25 @@ export const ELEMENT_DISCUSSION =
 export default class DiscussionArea {
   static #ELEMENT_POST_TEXTAREA = document.querySelector('#post-textarea');
   static #ELEMENT_TIMESTAMP_SPAN = document.querySelector('#timestamp-span');
+  static #ELEMENT_DISCUSSION = document.querySelector('#discussion');
+  static #ID_DISCUSSION_CONTAINER = 'discussion-comments';
   #lecture;
   #manager;
   #currentTimeMs;
   #currentRootCommentElements;
   #nearestComments;
+  #scrollContainer;
+  #discussionCommentsDiv;
 
   /**
    * Creates a `DiscussionArea` for a `lecture`.
    */
   constructor(lecture) {
     this.#lecture = lecture;
+    this.#scrollContainer = new ScrollContainer();
+    this.#scrollContainer.id = DiscussionArea.#ID_DISCUSSION_CONTAINER;
+    this.#discussionCommentsDiv = document.createElement('div');
```

Comment:
Can this be done outside of this constructor? We shouldn't be creating elements in the constructor? If this can only be done in a constructor can it go in ScrollContainer's constructor?

Reply:
We could create a factory method for creating a scroll container with a nested div.

Reply:
yes, although i'll need a reference to the `div` for adding things

Reply:
moved this to i`nitialzie`